setup/ubuntu/bootstrap.sh

```diff
@@ -157,6 +157,10 @@ apt-get install -y libffi-dev libssl-dev
 # MySQL dependencies:
 apt-get install -y libmysqlclient-dev
 
+# Microsoft SQL Server dependencies (`sudo` required):
+sudo apt-get install -y freetds-dev
+sudo pip install pymssql==2.1.1
```

Comment:
[Lines 7-16](https://github.com/joeharris76/redash/blob/master/setup/ubuntu/bootstrap.sh#L7-L16) verify that we run as root (sudo).